ebos/equil/initstateequil.hh

```diff
@@ -1899,6 +1899,10 @@ private:
 
             Details::verticalExtent(cells, cellZMinMax_, comm, vspan);
 
+            const auto acc = rec[r].initializationTargetAccuracy();
+            if (acc > 0)
+                throw std::runtime_error("Cannot initialise: Positive item 9 is not supported");
```

Comment:
Could you add a little more diagnostic information to this message, please?  Something along the lines of
```C++
throw std::runtime_error {
    "Cannot initialise model: Positive item 9 is not supported "
    "in EQUIL keyword, record " + std::to_string(r + 1)
};
```